diff --git a/node_modules/oidc-provider/lib/shared/token_auth.js b/node_modules/oidc-provider/lib/shared/token_auth.js
index 1587849..bbb14c1 100644
--- a/node_modules/oidc-provider/lib/shared/token_auth.js
+++ b/node_modules/oidc-provider/lib/shared/token_auth.js
@@ -87,7 +87,7 @@ module.exports = function tokenAuth(provider, endpoint) {
           if (clientId !== undefined && ctx.oidc.authorization.clientId !== clientId) {
             throw new InvalidRequest('mismatch in body and authorization client ids');
           }
-          if (clientSecret !== undefined) throw new InvalidRequest('client authentication must only be provided using one mechanism');
+          // if (clientSecret !== undefined) throw new InvalidRequest('client authentication must only be provided using one mechanism');
         } else if (clientId !== undefined) {
           ctx.oidc.authorization.clientId = clientId;
           ctx.oidc.authorization.methods = ['client_secret_post', 'none', 'tls_client_auth', 'self_signed_tls_client_auth'];
